### ISSUE TITLE
Give the "Add Podcast" item on the My Podcasts tab a proper label for the screen reader

### ISSUE THREAD

**Issue by MarcoZehe** [mozilla/high-fidelity#57]
The screen reader built into Firefox OS is used to give blind people access to the operating system and its apps via synthesized speech. An introduction can be found [here](https://wiki.mozilla.org/Accessibility/Mobile/ScreenReader).

The Add Podcast button at the top of the my screen's My Podcasts tab is missing a screen reader label. To reproduce:
1. Run Podcasts, and make sure you already have at least one item in the list.
2. Turn on the screen reader via quick toggle, as shown in the overview linked above.
3. Touch the item that would add a podcast.

Expected: Screen reader should say "Add Podcast" followed by "link".

Actual: Screen reader will only say "Link header", which indicates that it found there _is_ a link, just not what it is supposed to do.

Possible solution: I suspect this is an icon font or something similar. Give the a element an aria-label attribute whose value is a localized string like "Add Podcast".
